core/src/main/java/com/radixdlt/consensus/Vertex.java

```diff
@@ -114,9 +114,10 @@ public final class Vertex {
   @DsonOutput(Output.ALL)
   private final List<byte[]> transactions;
 
+  // TODO: rename this JSON property (or remove it entirely?)
```

Comment:
I guess we're not renaming it now for backwards compatibility of the hotfix?
Could we maybe add a little comment explaining when this flag would be true? 👍

Reply:
yup, not including it for compatibility